framework/src/modules/network/defaults/config.js

```diff
@@ -52,27 +52,63 @@ const defaultConfig = {
 		blacklistedPeers: {
 			type: 'array',
 			items: {
-				type: 'string',
-				format: 'ip',
+				type: 'object',
+				properties: {
+					ip: {
+						type: 'string',
+						format: 'ipOrFQDN',
```

Comment:
wasn't blackList suppose to be list of IPs not including port?

Reply:
think to have a consistent definition of peer makes it clear for usability and configuration.

Reply:
FQDN cannot be used for these things.
Seed peers are only used at the beginning, therefore we converted to the ip at the beginning.
However, other ip are the runtime validation, and we should not use the DNS resolver.

Reply:
Yes, I have reverted the changes to use only IP for blacklist. For the seedPeers makes sense to use FQDN because we have seed peers by default in the config when someone starts a node whereas for the fixed and white list are optional lists that will be used by the user for their use-cases.